src/AppBundle/Controller/MembershipController.php

```diff
@@ -139,12 +142,43 @@ public function pinInterestsAction(Request $request): Response
     }
 
     /**
+     * This action enables a new user to follow his/her committees.
+     *
      * @Route("/inscription/choisir-des-comites", name="app_membership_choose_nearby_committee")
      * @Method("GET|POST")
      */
-    public function chooseNearByCommitteeAction(): Response
+    public function chooseNearByCommitteeAction(Request $request): Response
     {
-        return new Response('TO BE IMPLEMENTED');
+        $membershipUtils = $this->get('app.membership_utils');
+
+        if (!$id = $membershipUtils->getNewAdherentId()) {
+            throw new AccessDeniedHttpException();
+        }
+
+        $manager = $this->getDoctrine()->getManager();
+
+        if (!$adherent = $manager->getRepository(Adherent::class)->find($id)) {
+            throw new NotFoundHttpException();
+        }
+
+        $form = $this->createForm(MembershipChooseNearbyCommitteeType::class);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            foreach ($form->get('committees')->getData() as $row) {
+                $membership = CommitteeMembership::createForAdherent($adherent, $row);
+                $manager->persist($membership);
+            }
+
+            $manager->flush();
+            $this->addFlash('info', $this->get('translator')->trans('adherent.registration.success'));
```

Comment:
Sorry, this is the user story requirement. We have to talk and be agree about that.